wee_slack.py

```diff
@@ -622,13 +622,13 @@ def buffer_prnt(self, user='unknown_user', message='no message', time=0):
         elif message.find(self.server.nick.encode('utf-8')) > -1:
             tags = ",notify_highlight,log1"
         elif user != self.server.nick and self.name in self.server.users:
-            tags = ",notify_private,notify_message,log1"
+            tags = ",notify_private,notify_message,log1,notify_highlight,irc_privmsg"
```

Comment:
Is the notify_highlight needed? My understanding is notify_private is a superset of that.